BraintreeCore/src/main/java/com/braintreepayments/api/BraintreeClient.java

```diff
@@ -232,17 +293,14 @@ boolean canPerformBrowserSwitch(FragmentActivity activity, @BraintreeRequestCode
     }
 
     void reportCrash() {
-        analyticsClient.reportCrash(applicationContext, sessionId, integrationType);
+        Authorization authorization = authorizationLoader.getAuthorizationFromCache();
+        analyticsClient.reportCrash(applicationContext, sessionId, integrationType, authorization);
     }
 
     static boolean isAnalyticsEnabled(Configuration configuration) {
         return configuration != null && configuration.isAnalyticsEnabled();
     }
 
-    Authorization getAuthorization() {
```

Comment:
Replaced this with an asynchronous variant.

Reply:
We do use this in Drop-in now to set the authorization in the launch intent extras. I think we might want to keep this until we confirm the use case in Drop-in

Reply:
Drop-in should be able to use the async version too. This would initially be `null` when a `ClientTokenProvider` is used and would invite race conditions in DropIn if the sync version is relied on.